buildbot-conf/cordova.conf

```diff
@@ -300,10 +300,13 @@ c['builders'].extend([
     BuilderConfig(name='cordova-ios',            slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'ios',          'slaveos': OSX}),
     BuilderConfig(name='cordova-android-osx',    slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'android',      'slaveos': OSX}),
     BuilderConfig(name='cordova-blackberry-osx', slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'blackberry10', 'slaveos': OSX}),
+    BuilderConfig(name='cordova-firefoxos-osx',  slavenames=OSX_SLAVES,     factory=cordova_plugins_all,     properties={'platform': 'firefoxos',    'slaveos': OSX}),
     BuilderConfig(name='cordova-windows',        slavenames=WINDOWS_SLAVES, factory=cordova_plugins_windows, properties={'platform': 'windows',      'slaveos': WINDOWS}),
     BuilderConfig(name='cordova-wp8',            slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'wp8',          'slaveos': WINDOWS}),
     BuilderConfig(name='cordova-android-win',    slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'android',      'slaveos': WINDOWS}),
     BuilderConfig(name='cordova-blackberry-win', slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'blackberry10', 'slaveos': WINDOWS}),
+    BuilderConfig(name='cordova-firefoxos-win',  slavenames=WINDOWS_SLAVES, factory=cordova_plugins_all,     properties={'platform': 'firefoxos',    'slaveos': WINDOWS}),
```

Comment:
Please add the new builders to the scheduler on [line 315](https://github.com/apache/cordova-medic/blob/master/buildbot-conf/cordova.conf#L315).